crates/tauri-bundler/src/utils/http_utils.rs

```diff
@@ -47,30 +49,36 @@ fn generate_github_alternative_url(url: &str) -> Option<(ureq::Agent, String)> {
 
   generate_github_mirror_url_from_template(url)
     .or_else(|| generate_github_mirror_url_from_base(url))
-    .map(|alt_url| (ureq::agent(), alt_url))
+    .map(|alt_url| {
+      (
+        ureq::Agent::config_builder()
+          .user_agent(BUNDLER_USER_AGENT)
+          .build()
+          .into(),
+        alt_url,
+      )
+    })
 }
 
 fn create_agent_and_url(url: &str) -> (ureq::Agent, String) {
   generate_github_alternative_url(url).unwrap_or((base_ureq_agent(), url.to_owned()))
 }
 
 pub(crate) fn base_ureq_agent() -> ureq::Agent {
+  let mut config_builder = ureq::Agent::config_builder()
```

Comment:
```suggestion
  #[allow(unused_mut)]
  let mut config_builder = ureq::Agent::config_builder()
```

Probably doesn't really matter though

Reply:
hmm right, didn't catch that because the feature is enabled by default